README.md

```diff
@@ -1,65 +1,72 @@
 # eProsima project documentation index
```

Comment:
I'd put a readme on how to use the repo. Also, add the license from fastdds-docs